src/views/nads/form/components/NetworkTypeParameters/NetworkTypeParameters.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+
+import { isEmpty } from '@utils/utils';
+
+import { networkTypeComponentMapper } from './utils/constants';
+import { ParametersComponentProps } from './utils/types';
+
+type NetworkTypeParametersProps = {
+  networkType: string;
+} & ParametersComponentProps;
+
+const NetworkTypeParameters: FC<NetworkTypeParametersProps> = ({
+  control,
+  networkType,
+  register,
+}) => {
+  const ParametersComponent = networkTypeComponentMapper[networkType];
```

Comment:
maybe ?. or use proxy on networkTypeComponentMapper

Reply:
I would better suggest to provide/create and use more specific type for `networkType` prop, more specific than `string`, and then we are good.

Reply:
type wont guard you on runtime. type is good for development